pkg/eval/platform/platform.go

```diff
@@ -25,5 +25,5 @@ import (
 
 var Ns = eval.Ns{
 	"arch": vars.NewReadOnly(runtime.GOARCH),
-	"os": vars.NewReadOnly(runtime.GOOS),
+	"os":   vars.NewReadOnly(runtime.GOOS),
```

Comment:
I have fixed the format in the latest commit, please rebase.

In case you haven't seen it, please use `goimports` to format code and configure your editor to run this automatically upon save (https://github.com/elves/elvish/blob/master/CONTRIBUTING.md#formatting-source-files).

Reply:
I installed `goimports` several months ago but haven't developed the "muscle memory" to run it when needed. I'm not a huge fan of running such tools in my editor context. I prefer git hooks or a `make style` target.